website/pages/docs/theming/component-style.mdx

```diff
@@ -132,10 +132,60 @@ const theme = extendTheme({
 })
 ```
 
+**And that's it!** You can use your new Button along with its custom variants throughout your app. But what if we want to create a custom component that's not part of Chakra UI? Let's use the following design spec for a Card component:
+
+<Img mt="10" mb="12" src="/card-spec.png" />
+
+Here's a contrived implementation of the design:
+
+```jsx live=false
+const Card = {
+  // The styles all Cards have in common
+  baseStyle: {
+    display: "flex",
+    flexDirection: "column",
+    background: "white",
+    alignItems: "center",
+    gap: 6
+  },
+  // Two variants: rounded and smooth
+  variants: {
+    rounded: {
+      padding: "32px",
+      borderRadius: "12px",
```

Comment:
What do you think about encouraging the reader to use theme tokens to keep their components in sync with the design system?

Reply:
That makes total sense! Thanks @TimKolberger